kolibri/core/assets/src/vue/exercise-attempts/answer-icon.vue

```diff
@@ -1,9 +1,9 @@
 <template>
 
   <div>
-    <svg v-if="isRight && !isHint" src="./icons/right.svg" class="yes" :class="{ yay: success }"></svg>
-    <svg v-if="!isRight && !isHint" src="./icons/wrong.svg" class="no" :class="{ yay: success }"></svg>
-    <svg v-if="isHint" src="./icons/hint.svg" class="no" :class="{ yay: success }"></svg>
+    <svg      v-if="answer === 'right'" src="./icons/right.svg" class="yes" :class="{ yay: success }"/>
+    <svg v-else-if="answer === 'wrong'" src="./icons/wrong.svg" class="no"  :class="{ yay: success }"/>
+    <svg v-else-if="answer === 'hint'"  src="./icons/hint.svg"  class="no"  :class="{ yay: success }"/>
```

Comment:
thanks - yeah, agreed it would be better to make these into constants, but they're used in very few places. (It's not an API constant)